src/sentry/static/sentry/app/icons/iconActivity.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {IconProps} from 'app/types/iconProps';
```

Comment:
Tiny nit, but we usually group our imports, and have blank lines between groups. In this case it would be 3rd party, then app.

So

```jsx
import React from 'react';

import {IconProps} from 'app/types/iconProps';
import theme from 'app/utils/theme';
```

Reply:
Np - I'll fix it